infrateam_first_responder.md

```diff
@@ -85,22 +85,22 @@ Status check from the server (ssh into the prod server for that project and then
 - Workflow server rails: `curl -i https://workflow-service-prod.stanford.edu/status/all`
 - Suri: `curl -i https://sul-suri-prod.stanford.edu/status/all`
 
-#### Required Additional Deploys
+### Required Additional Deploys
 
 There are applications that need to be deployed separately (i.e., not using `sdr-deploy`).
 
-##### Ruby Projects with non-standard Capistrano targets
+#### Ruby Projects with non-standard Capistrano targets
 
 * **hydra_etd `uat` environment**: deploy via `cap uat deploy` in `hydra-etd`
 * **sul-pub `cap-dev` enviroment**: deploy via `cap cap-dev deploy` in `sul_pub` (note all [ENV values](https://github.com/sul-dlss/sul_pub/tree/main/config/deploy))
 
-##### Cloud Projects
+#### Cloud Projects
 As of August 2021, deployment of cloud projects (Sinopia and DLME) happens via CircleCI, not manually.  You will need to make releases, however.
 
-* **Sinopia apps**:  Deployed when a new release is created.  Note:  the sinopia_editor project requires bumping the version number and a couple other steps before tagging a new release, see Sinopia [Release Process](https://github.com/LD4P/sinopia_editor/blob/main/release_process.md) for details.
-* **dlme-transform**: Deployed when a new release is created. See DLME [README](https://github.com/sul-dlss/dlme-transform/#deploying) and [DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/dlme/operations-concerns.md#deployment-info) for details.
+* **Sinopia apps**:  Deployed when a new release is created. Note: the sinopia_editor project requires bumping the version number and a couple other steps before tagging a new release, see Sinopia [Release Process](https://github.com/LD4P/sinopia_editor/blob/main/release_process.md) for details.
+* **dlme-transform**: Deployed when a new release is created, which happens when a PR is merged to main (e.g. dependency updates). You can confirm this by looking for the successful "publish-latest" step completion https://circleci.com/gh/sul-dlss/dlme-transform. See DLME [README](https://github.com/sul-dlss/dlme-transform/#deploying) and [DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/dlme/operations-concerns.md#deployment-info) for details.
```

Comment:
I'm starting to feel like this section should just be removed. There is no release process any longer and it seems to be confusing to just be included here.

Reply:
i'd be ok with that for dlme-transform.  i'd still find the sinopia part useful, since some FR work is required there.

Reply:
I like having something here to remind us this exists and is our responsibility.  I will edit further.

Reply:
@jmartin-sul oh yeah, I didn't mean to include sinopia in that :-)

Reply:
I think there is a release process, it's just that it ends at publishing to docker hub.